src/storage/exec/GetPropNode.h

```diff
@@ -28,23 +28,16 @@ class GetTagPropNode : public QueryNode<VertexID> {
         , vertexCache_(vertexCache) {}
 
     kvstore::ResultCode execute(PartitionID partId, const VertexID& vId) override {
-        // todo(doodle): revert to previous behavior, here is an extra io, will be deleted asap
-        // check is vertex exists
-        std::unique_ptr<kvstore::KVIterator> iter;
-        auto prefix = NebulaKeyUtils::vertexPrefix(planContext_->vIdLen_, partId, vId);
```

Comment:
I think you could put them to another PR. And according to current behavior in graph we need to determine does the vertex exists, so after this PR we need to seek all tag in current space for this @vesoft-inc/graph-reviewers

Reply:
If graph need to determine does the vertex exists, could call fetch *, as for those tags don't care, pass kTag. 

As the example said, v1 don't have t1 and t2, now we fetch t1 and t2. Current logic you can't explain why storage return different result.